manifests/charts/gateway/templates/_helpers.tpl

```diff
@@ -30,6 +30,10 @@ app.kubernetes.io/name: {{ include "gateway.name" . }}
 
 {{- define "gateway.podLabels" -}}
 {{ include "gateway.selectorLabels" . }}
+app.kubernetes.io/name: {{ include "gateway.name" . }}
+{{- with .Values.revision }}
+istio.io/rev: {{ . | quote }}
```

Comment:
Should we also add app.kubernetes.io/version	(or use that instead of istio.io/rev ) ? 

Also would be nice if we added the K8S recommended labels consistently to all our templates - consistency is important.

Reply:
added in 281c532ee5 

and optional cleanup in 3ac9a50475

marking as resolved - please reopen anytime with new comments 

+1 consistency